osquery/tables/events/tests/linux/process_events_tests.cpp

```diff
@@ -165,6 +166,59 @@ TEST_F(ProcessEventsTests, exec_event_processing) {
   }
 }
 
+TEST_F(ProcessEventsTests, kill_syscall_event_processing) {
+  // clang-format off
+  const RawAuditEvent kSampleKillEvent = {
+    { 1300, "audit(1588703361.452:26860): arch=c000003e syscall=62 success=yes exit=0 a0=6334 a1=f a2=0 a3=7f8b95cbbcc0 items=0 ppid=6198 pid=6199 auid=1000 uid=1000 gid=1000 euid=1000 suid=1000 fsuid=1000 egid=1000 sgid=1000 fsgid=1000 tty=pts3 ses=5 comm=\"bash\" exe=\"/bin/bash\" key=226B696C6C73686F7422" },
```

Comment:
Are the three `kill` system calls all generating the same records? Is the 'syscall' field value the only difference between the different events that we could encounter?

Reply:
Thank you taking a look at the PR.

Since `KILL` is also a syscall (the other 2 as well), I believe they should present the same audit record type 1300 and with similar structure. I haven't explicitly tested `TKILL` and `TGKILL`. I will try to produce those audit events as well.